Windows build error with cmake

When using cmake for windows build, I see the following error:

75>CustomBuild:
         Error copying file "dist/foundationdb-6.1.0.tar.gz" to "C:/Dev/cygwin/home/fdb/build/packages/foundationdb-6.1.0.tar.gz".

Markus, can you please take a look?

I'm having a different issue when trying to build with cmake on Windows. I followed the instructions in the readme (had to also install python, cf #1243), but now I'm getting the same build error whether via msbuild or devenv:

A sample of the errors:
```
33>C:\Users\chevalier\Downloads\boost_1_67_0\boost_1_67_0\boost/config/detail/select_stdlib_config.hpp(18): fatal error C1083: Cannot open include file: 'cstddef': No such file or directory
32>C:\Data\Git\GitHub\foundationdb\flow/Hash3.h(5): fatal error C1083: Cannot open include file: 'stdint.h': No such file or directory
32>C:\Data\Git\GitHub\foundationdb\flow/flow.h(29): fatal error C1083: Cannot open include file: 'vector': No such file or directory
``

Are you building from master? or a different branch? Am I missing another dependency to be able to build?

--- a/bindings/python/CMakeLists.txt
+++ b/bindings/python/CMakeLists.txt
@@ -62,7 +62,7 @@ endif()
 set(package_file_name foundationdb-${FDB_VERSION}.tar.gz)
 set(package_file ${CMAKE_BINARY_DIR}/packages/${package_file_name})
 add_custom_command(OUTPUT ${package_file}
-  COMMAND $<TARGET_FILE:Python::Interpreter> setup.py sdist &&
+  COMMAND $<TARGET_FILE:Python::Interpreter> setup.py sdist --formats=gztar &&
           ${CMAKE_COMMAND} -E copy dist/${package_file_name} ${package_file}
   WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
   COMMENT "Create Python sdist package")

